src/services/utils/vanilla.ts

```diff
@@ -0,0 +1,23 @@
+import Axios from 'axios';
+
+export class Vanilla extends MinecraftFlavor {
+  MANIFEST: string =
+    'https://launchermeta.mojang.com/mc/game/version_manifest.json';
+
+  constructor(version: string) {
+    super(version);
+  }
+
+  async getServerUrl(): Promise<string> {
+    const manifestResponse = await Axios.get(this.MANIFEST);
+    const versions = manifestResponse.data.versions;
+    let url = '';
+    for (let version of versions) {
+      if (version.id == this.version) url = version.url;
```

Comment:
It got formatted, but for some reason prettier didn't add the brackets. I wonder if it's using a different formatter. Anyway don't worry about it too much, I'll figure out why it's being weird.